ci-operator/step-registry/fips-check/fips-or-die/fips-check-fips-or-die-commands.sh

```diff
@@ -111,6 +111,22 @@ images=("quay.io/openshift-qe-optional-operators/myapp:v1.16-1"
 set_proxy
 run_command "oc whoami"
 run_command "oc version -o yaml"
+
+version=`oc get clusterversion version -o=jsonpath="{.status.desired.version}"`
```

Comment:
Do not skip based on the clusterversion field.

Let's skip only if finding a node (worker or master) based on the arm64 architecture.

Also, what's blocking us from testing on P/Z, which images are not built for those architectures yet and where are their definitions and CI hosted?

If that's challenging to build for p/z yet, let's open a jira in the mulit-arch to track and discuss the images' build enablement and skip also if any node with p/z workers/masters is found.

Reply:
+1 - this field is not guaranteed  to be set. this is for the purpose of the user to indicate a desired transition to multi payload.